src/Services/Billing/BillingService.php

```diff
@@ -2,9 +2,67 @@
 
 namespace Vultr\VultrPhp\Services\Billing;
 
+use Vultr\VultrPhp\Services\VultrServiceException;
 use Vultr\VultrPhp\Services\VultrService;
+use Vultr\VultrPhp\Util\ListOptions;
 
 class BillingService extends VultrService
 {
+	private static ?array $cache_bill = null;
 
+	/**
+	 * @param $options - ListOptions - Interact via reference.
+	 * @throws BillingException
+	 * @return Bill[]
+	 */
+	public function getBills(?ListOptions &$options = null) : array
```

Comment:
getBillingHistory

Reply:
Your implementing the following functions here.
https://www.vultr.com/api/#tag/billing